pinax/stripe/tests/test_webhooks.py

```diff
@@ -113,7 +113,7 @@ def test_webhook_associated_with_stripe_account(self, TransferMock, StripeEventM
         connect_event_data = self.event_data.copy()
         stripe_account = "acct_123123123"
         # only difference is that we'll have a user_id value
-        connect_event_data["user_id"] = stripe_account
+        connect_event_data["account"] = stripe_account
```

Comment:
Does this work? Events definitely had `user_id` specifying the stripe account id when I implemented this. Great if it's changed as it's an awful key, but we need to be careful as to which version this became standard.

Reply:
For reference https://stripe.com/docs/upgrades#2017-05-25
Since stripe Connect support just landed in Pinax Stripe we can just document that is the minimum required API version.

Reply:
Added to docs.